azure-pipelines.yml

```diff
@@ -0,0 +1,36 @@
+pr:
+- main
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - main
+
+resources:
+  repositories:
+  - repository: buildimage
+    type: github
+    name: sonic-net/sonic-buildimage
+    endpoint: sonic-net
+    ref: master
+
+variables:
+  - template: /.azure-pipelines/template-variables.yml@buildimage
+  - template: /.azure-pipelines/azure-pipelines-repd-build-variables.yml@buildimage
+
+stages:
+- stage: Build
+  jobs:
+  - job: build
+    pool: sonicbld
+    timeoutInMinutes: 360
+    steps:
+    - checkout: self
+      clean: true
+    - bash: |
+        set -ex
+        make sonic ENABLE_DOCKER_BASE_PULL=y SONIC_BUILD_JOBS=$(nproc)
+        cp build/sonic-buildimage/target/docker-sonic-vpp.gz $(Build.ArtifactStagingDirectory)
```

Comment:
Please also copy build/sonic-buildimage/target/sonic-vpp.img.gz to ArtifactStaging directory